lib/tasks/auth_source_external.rake

```diff
@@ -0,0 +1,23 @@
+desc 'Create Auth Source External'
+namespace :auth_source_external do
+  desc 'create auth source external'
+  task :create => :environment do
+    User.as_anonymous_admin do
+      external_name = Setting[:authorize_login_delegation_auth_source_user_autocreate]
+      return unless external_name.present?
+
+      auth_source_external = AuthSourceExternal.find_or_create_by!(name: external_name)
+
+      Taxonomy.types.each do |taxonomy|
+        tax_association = taxonomy.to_s.downcase.pluralize
+
+        default_tax_name = ENV.fetch("SEED_#{taxonomy.to_s.upcase}", "Default #{taxonomy}")
```

Comment:
This doesn't run as part of the seed so this env variable doesn't make much sense. I would either go with arguments for the task or skip the taxonomy assignment here completely and leaving it for the api.

Reply:
Let's skip it from here and allow the admin to add it form the API.